src/main/java/com/google/solutions/ml/api/vision/PubSubNotificationToGCSUriDoFn.java

```diff
@@ -45,7 +45,7 @@ public void processElement(ProcessContext c) {
     PubsubMessage message = c.element();
     String eventType = message.getAttribute("eventType");
     if (!Objects.equals(eventType, "OBJECT_FINALIZE")) {
```

Comment:
here, and on line 60, it'd be nice to have these conditionals in `.filter`s elevated to the main pipeline before this dofn.

I'm also not clear why you're running `.inc()` (ln 51) on a file you may not process.

Reply:
oh I see, you have a separate rejectedfiles counter. disregard the 2nd part

Reply:
Addressed the filtering comment.